authlib/consts.py

```diff
@@ -1,4 +1,12 @@
 name = 'authlib'
-version = 'plt.2893.0'
+version = 'plt.2865.0'
 author = 'Quartic.ai Engineering Team'
 homepage = 'https://github.com/Quarticai/authlib'
+
+default_user_agent = '{}/{} (+{})'.format(name, version, homepage)
+
+default_json_headers = [
+    ('Content-Type', 'application/json'),
+    ('Cache-Control', 'no-store'),
+    ('Pragma', 'no-cache'),
+]
```

Comment:
wha are these changes for?

Reply:
These default user agent and default json header is being used inside the integration class of base client in master branch. But these lines were removed in develop branch and was throwing error so I added them back

Reply:
ohh okay 👍